code/vignette/0.-loading-flat-files.Rmd

```diff
@@ -68,6 +68,9 @@ benefit from more compression than flat files do), and they're faster to read
 back into R.
 
 ```{r saving}
+if(!dir.exists(paste0(outdir, "/processed_data"))){
+  dir.create(paste0(outdir, "/processed_data"))
+}
 saveRDS(atomxdata$counts, file = paste0(outdir, "/processed_data/counts_unfiltered.RDS"))
 saveRDS(atomxdata$negcounts, file = paste0(outdir, "/processed_data/negcounts_unfiltered.RDS"))
 saveRDS(atomxdata$falsecounts, file = paste0(outdir, "/processed_data/falsecounts_unfiltered.RDS"))
```

Comment:
@maddygriz , question re: this:
```
Now we have data in 3 locations:

- The RDS files we just created
- The flat files we began with
- The original data on AtoMx

This is unnecessary duplication. Once you've confirmed this script has run successfully,
you can safely delete the flat files - you won't be using them again. 
If you do need to go back to them, you can always export them from AtoMx again:
AtoMx acts as a persistent, authoritative source of unmodified data.
```

What do you think of these recommendations to avoid storing duplicated data:
1. throw out the flat files and re-export from atomx if needed
2. keep the flat files, and throw out your derived RDS files once you're done with active analysis.